stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/app.py

```diff
@@ -28,52 +24,15 @@
 settings = ElasticsearchSettings()
 session = Session.create_from_settings(settings)
 
-
-@attr.s
-class FixedSortExtension(SortExtension):
-    """SortExtension class fixed with correct paths, removing extra forward-slash."""
-
-    conformance_classes: List[str] = attr.ib(
-        factory=lambda: ["https://api.stacspec.org/v1.0.0-beta.4/item-search#sort"]
-    )
-
-
-@attr.s
-class FixedFilterExtension(FilterExtension):
-    """FilterExtension class fixed with correct paths, removing extra forward-slash."""
-
-    conformance_classes: List[str] = attr.ib(
-        default=[
-            "https://api.stacspec.org/v1.0.0-rc.1/item-search#filter",
-            "http://www.opengis.net/spec/ogcapi-features-3/1.0/conf/filter",
-            "http://www.opengis.net/spec/ogcapi-features-3/1.0/conf/features-filter",
-            "http://www.opengis.net/spec/cql2/1.0/conf/cql2-text",
-            "http://www.opengis.net/spec/cql2/1.0/conf/cql2-json",
```

Comment:
@philvarner added a link for cql2-json which doesn't seem to be in the stac-fastapi defaults? https://github.com/stac-utils/stac-fastapi/blob/925a9bbc686dcb1f17808719ab9863c08b92ea36/stac_fastapi/extensions/stac_fastapi/extensions/core/filter/filter.py#L73-L81

Reply:
that's important to have here.  PC (which uses stac-fastapi) seems to have it https://planetarycomputer.microsoft.com/api/stac/v1/ 

That seems to be a bug, as it's defined-but-unused here https://github.com/stac-utils/stac-fastapi/blob/925a9bbc686dcb1f17808719ab9863c08b92ea36/stac_fastapi/extensions/stac_fastapi/extensions/core/filter/filter.py#L31

Reply:
Did you use cql2-json in this project and it maybe isn't used in the other stac-fastapi versions?

Reply:
no, pgstac supports both also. I think it was just an oversight when the conformance classes were refactored.

In fact, pgstac's primary support is for CQL2 JSON, and the CQL Text inputs are first translated to JSON before being passed to the backend.